templates/docs/shared/_footer.html

```diff
@@ -0,0 +1,40 @@
+<div class="p-strip is-dark l-docs__footer">
+  <div class="row">
+    <footer class="l-docs__subgrid">
+      <div class="l-docs__sidebar">
+        <p style="padding-left: 1.5rem">
```

Comment:
```suggestion
        <p>
```

Reply:
I followed an example from https://vanillaframework.io/docs/layouts/documentation

Reply:
Yes, that is the full-width site example. The one used here https://microk8s.io/docs

Reply:
It also breaks on medium screens
![image](https://github.com/user-attachments/assets/8be9ece1-ec25-4479-981d-fb1bab9b0fb5)